ui/codeView.js

```diff
@@ -2096,6 +2113,12 @@ function disable() {
             GLib.source_remove(PANEL_WAITER);
             PANEL_WAITER = 0;
         });
+
+        if (CLUBHOUSE_TOOLTIP_HANDLER) {
+            const clubhouse = getClubhouseAppIcon();
+            destroyInfoPopup.bind(clubhouse)(CLUBHOUSE_TOOLTIP_HANDLER);
```

Comment:
Should probably check if the icon instance exists before attempting to invoke `destroyInfoPopup` on it.

Reply:
Actually this should always be the case when `CLUBHOUSE_TOOLTIP_HANDLER` is set but I guess it wouldnt hurt to double check and maybe even warn if it doesn't.

Reply:
I agree, I've added the check just before the call to ensure.